examples/client.go

```diff
@@ -282,7 +282,12 @@ func main() {
 		}
 		currInstances := int32(len(live))
 		fmt.Println("Current num of instances: ", currInstances)
-		resp, err := r.AddInstances(aurora.InstanceKey{job.JobKey(), 0}, numOfInstances)
+		var instId int32
+		for k := range live{
```

Comment:
As discussed, cover the scenario when instId is not initialized, i.e. when there aren't any active task..